flow2ml/README.md

```diff
@@ -50,6 +50,8 @@ Applies the following augmentation operations to the images in data directory<br
     <li>flipping</li>
     <li>rotation</li>
     <li>shearing</li>
+    <li>inverting</li>
+    <li>Histogram Equalization</li>
```

Comment:
Same as above. When we already changes related to Histogram Equalization in main branch, why do we need these here.